src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -229,6 +229,10 @@ class DefaultStorageKeyAllocator
     static StorageKeyName TSDefaultNTP() { return StorageKeyName::FromConst("g/ts/dntp"); }
     static StorageKeyName TSTimeZone() { return StorageKeyName::FromConst("g/ts/tz"); }
     static StorageKeyName TSDSTOffset() { return StorageKeyName::FromConst("g/ts/dsto"); }
+
+    // ICD Client Info
+    static StorageKeyName ICDClientInfo(size_t index) { return StorageKeyName::Formatted("g/su/%x", static_cast<unsigned>(index)); }
+    static StorageKeyName ICDClientInfoMaxCount() { return StorageKeyName::Formatted("g/sum"); }
```

Comment:
Adding new `g/*` keys on clients is not OK.

Reply:
removed, add ICDStorageKeyDelegate to handle this issue